content-repo/extra-docs/releases/20.9.2.md

```diff
@@ -0,0 +1,1163 @@
+
+##### BMCHelixRemedyforce - Incoming Mapper  
+Maps incoming BMC Helix Remdey force incidents fields.
+#### IncidentFields
+- **Bmc Remedyforce Attachment(s)**
+- **Bmc Remedyforce Broadcast**
+- **Bmc Remedyforce Category**
+- **Bmc Remedyforce Client Account**
+- **Bmc Remedyforce Client Name**
+- **Bmc Remedyforce Closed Date**
+- **Bmc Remedyforce Configuration Item / Asset**
+- **Bmc Remedyforce Created Date**
+- **Bmc Remedyforce Description**
+- **Bmc Remedyforce Due Date**
+- **Bmc Remedyforce ID**
+- **Bmc Remedyforce Impact**
+- **Bmc Remedyforce Last Modified Date**
+- **Bmc Remedyforce Note(s)**
+- **Bmc Remedyforce Opened Date**
+- **Bmc Remedyforce Outage End**
+- **Bmc Remedyforce Outage Start**
+- **Bmc Remedyforce Queue**
+- **Bmc Remedyforce Request Definition**
+- **Bmc Remedyforce Resolution**
+- **Bmc Remedyforce Responded Date**
+- **Bmc Remedyforce Service**
+- **Bmc Remedyforce Service Offering**
+- **Bmc Remedyforce Service Request**
+- **Bmc Remedyforce Staff**
+- **Bmc Remedyforce Status**
+- **Bmc Remedyforce Template**
+- **Bmc Remedyforce Urgency**
+#### IncidentTypes
+- **Bmc Remedyforce Incident**
+- **Bmc Remedyforce Service Request**
+#### Integrations
+##### BMC Helix Remedyforce  
+BMC Helix Remedyforce integration enables customers to create/update service requests and incidents, update statuses, and resolve service requests and incidents with customer notes. This integration exposes standard ticketing capabilities that can be utilized as part of automation & orchestration.
+#### Layouts
+- **Bmc Remedyforce Incident - Summary**
+- **Bmc Remedyforce Service Request - Summary**
+
+#### Scripts
+##### BMCHelixRemedyforceCreateIncident  
+This script is used to simplify the process of creating the incident in BMC Helix Remedyforce. The Script will consider the ID over the name of the argument when both are provided. Example: client_id is considered when both client_id and client_user_name are provided.
+##### BMCHelixRemedyforceCreateServiceRequest  
+This script is used to simplify the process of creating a service request in BMC Helix Remedyforce. The script will consider the ID over the name of the argument when both are provided. Example: client_id is considered when both client_id and client_user_name are provided.
+
+
+---
+
+### New: Bonusly Pack v1.0.0
+#### Integrations
+##### Bonusly  
+The Bonusly integration is used to interact with the Bonusly platform through the API. Bonusly is an employee recognition platform which enterprises use to for employee recognition.
+#### Playbooks
+##### Bonusly - AutoGratitude  
+AutoGratitude is a playbook to give back a positive gratitude to security engineers and developers when they successfully complete an SLA
+#### Scripts
+##### IncOwnerToBonuslyUser  
+This script gets the email address of the incident owner and then returns the incident owner username in Bonusly.
+
+
+---
+
+### New: CaseManagement-Generic Pack v1.0.0
```

Comment:
Add the following pack description one line below:
Beta Pack built by the Cortex Customer Success Team to provide quick and valuable deployment of XSOAR for the main use of Case Management.